docs/conf.py

```diff
@@ -103,22 +110,6 @@
 
 
 # -- Options for LaTeX output ------------------------------------------------
```

Comment:
Projects should not have any options for LaTex output...

Reply:
We have some options built in, but I can remove that section. I think this was just boiler plate stuff.